docs/_getting-started/index.md

```diff
@@ -4,89 +4,202 @@ permalink: docs/getting-started
+These include the main Ethereum network, [a private network](getting-started/private-net) you  create, 
+and three test networks that use different consensus algorithms:
 
-_[Read this guide](./interface/managing-your-accounts) for more details on importing
-existing Ethereum accounts and other uses of the `account` command._
+-   **Ropsten**: Proof-of-work test network
+-   **Rinkeby**: Proof-of-authority test network
+-   **Görli**: Proof-of-authority test network
+
+For this guide, we use the Görli network.
 
 ### Sync modes
 
-Running Geth starts an Ethereum node that can join any existing network, or create a new
-one. You can start Geth in one of three different sync modes using the `--syncmode "{mode}"`
+You can start Geth in one of three different sync modes using the `--syncmode "<mode>"`
 argument that determines what sort of node it is in the network.
 
 These are:
 
--   **Full**: Downloads all blocks (including headers, transactions and receipts) and
+-   **Full**: Downloads all blocks (including headers, transactions, and receipts) and
     generates the state of the blockchain incrementally by executing every block.
 -   **Fast** (Default): Downloads all blocks (including headers, transactions and
     receipts), verifies all headers, and downloads the state and verifies it against the
     headers.
 -   **Light**: Downloads all block headers, block data, and verifies some randomly.
 
-For example:
+For this tutorial, we use a `light` sync:
+
+## Start Clef
+
+Start Clef, setting the keystore and chain id (goerli is 5) for the network we want to connect to:
+
+```shell
+clef --keystore <GETH_LOCATION>/keystore --chainid 5
+```
+
+To begin with, you see errors about a missing keystore, and we fix that soon.
+
+## Start Geth
+
+The command below also enables the [Geth RPC interface](clef/tutorial) 
+(which we cover below), and sets Clef as the transaction signer.
+
+```shell
+geth --goerli --syncmode "light" --rpc --signer=<CLEF_LOCATION>/clef.ipc
+```
+
+## Get ETH
+
+Unless you have Ether in another account on the Görli network, you can use a
+[faucet](https://goerli-faucet.slock.it/) to send ETH to one of your new account addresses to use for this guide.
+
+## Connect to Geth with IPC or RPC
+
+You can interact with Geth in two ways: Directly with the node using the JavaScript
+console over IPC, or connecting to the node remotely over HTTP using RPC.
+
+IPC allows you to do more, especially when it comes to creating and interacting
+with accounts, but you need direct access to the node.
+
+RPC allows remote applications to access your node but has limitations and security
+considerations, and by default only allows access to methods in the `eth` and `shh`
+namespaces. Find out how to override this setting [in the RPC docs](rpc/server#http-server). 
+
+## Using IPC
+
+### Connect to console
+
+You can connect to the IPC console in two ways. First, when you start the node:
```

Comment:
I think we shouldn't even have the `geth console` way in the guide anymore. It's kind-of redundant and just means more flags need to be passed. `geth attach` is better.